manifests/addons/values-kiali.yaml

```diff
@@ -7,5 +7,7 @@ deployment:
   accessible_namespaces:
   - '**'
   ingress_enabled: false
+  hpa:
+    api_version: autoscaling/v2
```

Comment:
This one is a bit trickier since it ends up as plain YAML.. but this will break users. Maybe we should be more conservative for now and use v2beta2 for a bit here?

Reply:
done